testing/engine/chains.go

```diff
@@ -154,7 +154,7 @@ SecRule REQUEST_HEADERS "@rx attack21" \
     "msg:'Sub Chain Rule',\
     logdata:'FoundSubChain21 %{MATCHED_VAR} in %{MATCHED_VAR_NAME}',\
     chain"
-    SecRule MATCHED_VAR_NAME "@rx MATCHED_VARS_NAMES:REQUEST_HEADERS:Host" \
```

Comment:
Please triple-check this, as my implementation unified name collections, I found inconsistency between chains and setvar test

https://github.com/corazawaf/coraza/blob/v3/dev/testing/engine/setvar.go#L38

Previously, MATCHED_VAR_NAME was being populated in a way that for ARGS_NAMES, but for MATCHED_VAR_NAMES it had the variable name MATCHED_VAR_NAME suffixed with its value, REQUEST_HEADERS:Host. This doesn't seem consistent to me, and names are presumably supposed to just be a flat list of names. Can anyone verify the behavior of MATCHED_VAR_NAME + MATCHED_VAR_NAMES?

Reply:
Ping @piyushroshan @fzipi @jptosso  and cc @dune73 in case got an insight.

Reply:
You are right. Previous was incorrect.